src/Uno.UWP/UI/Core/CoreWindow.skia.cs

```diff
@@ -6,19 +8,40 @@ namespace Windows.UI.Core
 {
 	public partial class CoreWindow
 	{
+		internal event TypedEventHandler<CoreWindow, KeyEventArgs>? NativeKeyDownReceived;
+		internal event TypedEventHandler<CoreWindow, KeyEventArgs>? NativeKeyUpReceived;
+
 		internal bool IsNativeElement(object content)
-			=> _coreWindowExtension.IsNativeElement(content);
+			=> _coreWindowExtension!.IsNativeElement(content);
 
 		internal void AttachNativeElement(object owner, object content)
-			=> _coreWindowExtension.AttachNativeElement(owner, content);
+			=> _coreWindowExtension!.AttachNativeElement(owner, content);
 
 		internal void DetachNativeElement(object owner, object content)
-			=> _coreWindowExtension.DetachNativeElement(owner, content);
+			=> _coreWindowExtension!.DetachNativeElement(owner, content);
 
 		internal void ArrangeNativeElement(object owner, object content, Rect arrangeRect)
-			=> _coreWindowExtension.ArrangeNativeElement(owner, content, arrangeRect);
+			=> _coreWindowExtension!.ArrangeNativeElement(owner, content, arrangeRect);
 
 		internal Size MeasureNativeElement(object owner, object content, Size size)
-			=> _coreWindowExtension.MeasureNativeElement(owner, content, size);
+			=> _coreWindowExtension!.MeasureNativeElement(owner, content, size);
+
+		void ICoreWindowEvents.RaiseNativeKeyDownReceived(KeyEventArgs args)
+		{
+			NativeKeyDownReceived?.Invoke(this, args);
+			if (!args.Handled)
```

Comment:
I think that during investigation for the issue, I found that even `Handled` events are still raised on `CoreWindow`:

See in Expected behavior of the issue description: 

> Interestingly CoreWindow.KeyDown/CoreWindow.KeyUp are raised even if UIElement.KeyDown/KeyUp are marked as Handled (and the args of the CoreWindow event reflect the Handled value from the bubbling phase.